scriptmodules/emulators/sdltrs.sh

```diff
@@ -10,46 +10,56 @@
 #
 
 rp_module_id="sdltrs"
-rp_module_desc="Radio Shack TRS-80 Model I/III/4/4P emulator"
+rp_module_desc="Radio Shack TRS-80 Model I/III/4/4P emulator (SDL2 version)"
 rp_module_help="ROM Extension: .dsk\n\nCopy your TRS-80 games to $romdir/trs-80\n\nCopy the required BIOS file level2.rom, level3.rom, level4.rom or level4p.rom to $biosdir"
 rp_module_section="exp"
-rp_module_flags="dispmanx !mali !kms"
+rp_module_licence="BSD https://gitlab.com/jengun/sdltrs/-/raw/master/LICENSE"
+rp_module_flags=""
 
 function depends_sdltrs() {
-    getDepends libsdl1.2-dev libxt-dev
+    getDepends libsdl2-dev libreadline-dev
 }
 
 function sources_sdltrs() {
-    gitPullOrClone "$md_build" https://github.com/RetroPie/sdltrs.git
+    gitPullOrClone "$md_build" https://gitlab.com/jengun/sdltrs.git sdl2
 }
 
 function build_sdltrs() {
-    cd src/linux
     make clean
+    ./autogen.sh
+    ./configure --prefix="$md_inst"
     make
-    md_ret_require="$md_build/src/linux/sdltrs"
+    md_ret_require="$md_build/sdl2trs"
 }
 
 function install_sdltrs() {
-    chmod 644 README
     md_ret_files=(
-        'src/linux/sdltrs'
-        'README'
+        'sdl2trs'
+        'README.md'
+        'LICENSE'
     )
 }
 
 function configure_sdltrs() {
     mkRomDir "trs-80"
 
-    addEmulator 1 "$md_id-model1" "trs-80" "$md_inst/sdltrs -model 1 -romfile $biosdir/level2.rom -showled -diskdir $romdir/trs-80 -disk0 %ROM%"
-    addEmulator 0 "$md_id-model3" "trs-80" "$md_inst/sdltrs -model 3 -romfile3 $biosdir/level3.rom -showled -diskdir $romdir/trs-80 -disk0 %ROM%"
-    addEmulator 0 "$md_id-model4" "trs-80" "$md_inst/sdltrs -model 4 -romfile3 $biosdir/level4.rom -showled -diskdir $romdir/trs-80 -disk0 %ROM%"
-    addEmulator 0 "$md_id-model4p" "trs-80" "$md_inst/sdltrs -model 4p -romfile4p $biosdir/level4p.rom -showled -diskdir $romdir/trs-80 -disk0 %ROM%"
+    addEmulator 1 "$md_id-model1" "trs-80" "$md_inst/sdl2trs -fullscreen -nomousepointer -model 1 -romfile $biosdir/level2.rom -showled -diskdir $romdir/trs-80 -disk0 %ROM%"
+    addEmulator 0 "$md_id-model3" "trs-80" "$md_inst/sdl2trs -fullscreen -nomousepointer -model 3 -romfile3 $biosdir/level3.rom -showled -diskdir $romdir/trs-80 -disk0 %ROM%"
+    addEmulator 0 "$md_id-model4" "trs-80" "$md_inst/sdl2trs -fullscreen -nomousepointer -model 4 -romfile3 $biosdir/level4.rom -showled -diskdir $romdir/trs-80 -disk0 %ROM%"
+    addEmulator 0 "$md_id-model4p" "trs-80" "$md_inst/sdl2trs -fullscreen -nomousepointer -model 4p -romfile4p $biosdir/level4p.rom -showled -diskdir $romdir/trs-80 -disk0 %ROM%"
```

Comment:
if adding more parameters, I think it might be a good time to dedupe this a bit - can use a variable or an array or so.

Reply:
Yes, that would clear things a bit.